fdb-record-layer-lucene/src/main/java/com/apple/foundationdb/record/lucene/LucenePartitioner.java

```diff
@@ -76,6 +89,8 @@
  */
 @API(API.Status.EXPERIMENTAL)
 public class LucenePartitioner {
+
```

Comment:
Should we confirm that the provided partitioning field is also defined as "sortable" in the index?

Reply:
I think that's a good idea, but I think it would be better to do that as a followup because it's a bit tricky to confirm that.